config.schema.json

```diff
@@ -221,6 +221,34 @@
     "type": "string",
     "description": "ERC-20 contract for wrapped SWTH"
   },
+  "market_banners": {
+    "type": "array",
+    "description": "List of market banners",
```

Comment:
Maybe describe where the market banners will be displayed (i.e. `List of banner information to be displayed over the TradingView chart for the required market`)